src/run.ts

```diff
@@ -177,14 +177,18 @@ type VersionOptions = {
   hasPublishScript?: boolean;
 };
 
+type RunVersionResponse = {
```

Comment:
nit: it sounds like a response from some HTTP-like call to me but it's just a return value of an async function:
```suggestion
type RunVersionResult = {
```

Reply:
done